pages/data-providers/index.jsx

```diff
@@ -64,11 +64,17 @@ const getHelperMessage = ({ created, duplicated, error }) => {
 
   if (created) {
     return {
-      message: (
+      helper: (
         <>
-          We found {created.name} under the provided address and added it to our
-          data provider collection. We will start harvesting as soon as we are
-          able to approve it.
+          <p>
+            We found {created.name} under the entered address and added it to
+            our data provider collection. We will start harvesting as soon as we
+            are able to approve it.
+          </p>
+          <p>
+            After the approval we will sent a confirmation email to{' '}
+            {created.email}.
+          </p>
```

Comment:
Sorry, I suggested this. I am thinking that this is semantically wrong. A paragraph cannot be descendant fo the paragraph, i.e. if we wrap the input into a paragraph, the only plain text should appear here. What do you think @Joozty?

```suggestion
          We found {created.name} under the entered address and added it to
          our data provider collection. As soon as we approve adding, we will
          start harvesting and sent a confirmation email to{' '}
          <Link
            href={`mailto:${created.email}`}
            title="the administrator email address"
          >
            {created.email}
          </Link>.
```

I hope, this is shorter.